.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,103 @@
+name: Integration Test
+
+on:
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      checks: write
+      pull-requests: write
+      id-token: write
+    env:
+      iam_role_to_assume: ${{ secrets.ROLE_ARN }}
+      device_farm_project_arn: ${{ secrets.DEVICE_FARM_PROJECT_ARN }}
+      device_farm_pool_arn: ${{ secrets.DEVICE_FARM_POOL_ARN }}
+      device_farm_test_spec_arn: ${{ secrets.DEVICE_FARM_TEST_SPEC_ARN }}
+      clickstream_app_id: ${{ secrets.CLICKSTREAM_APP_ID }}
+      clickstream_endpoint: ${{ secrets.CLICKSTREAM_ENDPOINT }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up JDK 8
+        uses: actions/setup-java@v4
+        with:
+          java-version: '8'
+          distribution: 'corretto'
+          cache: gradle
+      - name: Build SDK release aar file
+        run: |
+          echo ${{ github.event.pull_request.title }}
+          chmod +x gradlew
+          ./gradlew assembleRelease
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: 'corretto'
+          cache: gradle
+      - name: Build sample android app
+        run: |
+          git clone -b integration_test https://github.com/aws-samples/clickstream-sdk-samples
```

Comment:
Let's update the `build.gradle` to use local ARR for this test. Don't need another branch for the changes.

Reply:
fixed.